index.js

```diff
@@ -686,32 +686,47 @@ Client.prototype.publishersInfo = function (publishers, callback) {
   if (publishers.length === 0) return
 
   if (typeof this.batches === 'undefined') this.batches = 0
-  this.publishers = this.publishers ? this.publishers.concat(publishers) : underscore.clone(publishers)
+  if (!this.publishers) this.publishers = { requests: [], results: [], uniques: [] }
+  publishers.forEach((publisher) => {
+    if (this.publishers.uniques.indexOf(publisher) !== -1) return console.log('skipping ' + publisher)
```

Comment:
Please remove this console log or put it behind verbose param

Reply:
sorry!